### ISSUE TITLE
Back to top link is not visible

### ISSUE THREAD

**Issue by phudson-he** [archesproject/arches#12720]
**Note:** This issue was found using arches 7.6.x

This issue impacts the following WCAG requirements: [2.4.1 Bypass Blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks) / [2.4.7 Focus Visible](https://www.w3.org/TR/WCAG21/#focus-visible)

**Steps to Reproduce:**
- Given I am a user reliant on the keyboard and/or screen reader to navigate
- And I am using the keyboard and NVDA screen reader to navigate through any page in the system
- When I navigate to the bottom of the page
- Then the screen reader tells me that I have reached the 'Back to Top' link, but no link is visible or receives focus, and pressing carriage return does nothing.

**Expected Result:**
The user is able to view/click/tab to and action a 'Back to Top' link which scrolls to the top of the page. This is particularly useful when viewing long pages on smaller screens. Upon scrolling down the page, at a certain point, the link/button should appear.

**Actual Result:**
The 'Back to Top' link/button is not currently visible at all and is only announced by the screen reader. Selecting the link using enter/carriage return doesn't seem to have any effect either.

**Evidence:**
Scroll down a long page either by using your mouse, keyboard or screen reader, you will notice no 'Back to Top' button/link displays, even though if you look at the HTML, it is present in the DOM and is announced by a screen reader even though it is not visible.

**Comment by phudson-he** [comment]
Just fyi, my work on this has paused due to other priorities. I'll leave this issue here for now.

**Comment by phudson-he** [comment]
For now, it was decided to do this CSS change, until a better solution can be included...

`#backToTopBtn {
        display: none !important;
    }`
